### ISSUE TITLE
UI tests (Jemmy/Jellytools) of Netbeans Application with surefire

### ISSUE THREAD

**Issue by jvtrudel** [mojohaus/nbm-maven-plugin#38]
Netbeans platforms is executed using nbexec that take into account various configurations as Look and Feel and various java options like available memory. 

On the other hand, surefire directly execute test classes with a simple java call. Hence, any problem related nbexec configuration cannot be tested. 

Is it in the roadmap of this project to provide a testing mechanism using nbexec? Does anyone already encounter this kind of issue and found a workaround?

**Comment by ebarboni** [comment]
Hi, I'm not sure this will answer your question but it is possible to add add args and systemproperties to surefire configuration 
   <argLine>@{argLine} -J-Xmx12g -J-Xss2m -J-Xms12g  -J-Dsun.java2d.noddraw=true </argLine>
I don't know the args for changing le L&F but is should be possible I guess

**Comment by jvtrudel** [comment]
Yes indeed. But there seems to be options specific to nbexec. I don't know much about this launcher, could it be completely replaced by arLine options. My  attempts to add the look and feel by that method failed.
